tests/network/services.go

```diff
@@ -154,64 +148,55 @@ var _ = SIGDescribe("Services", func() {
 		})
 
 		Context("with a service matching the vmi exposed", func() {
-			var jobsToCleanup []func() error
+			var jobToCleanup func() error
 
 			BeforeEach(func() {
 				serviceName = "myservice"
 
 				service := buildServiceSpec(serviceName, servicePort, servicePort, selectorLabelKey, selectorLabelValue)
 				_, err := virtClient.CoreV1().Services(inboundVMI.Namespace).Create(service)
 				Expect(err).ToNot(HaveOccurred())
-
-				// reset the job cleanup functions list
-				jobsToCleanup = nil
```

Comment:
Makes sense to still reset the job.

Reply:
Why ? I never compare it . I just go ahead and use it.

Reply:
* In case the test fails before the cleanup job is initialized the previous one will be used.
* Seems more right to start from a clean page. Maybe it even makes more sense to nullify the `jobsToCleanup` in the `afterEach`.

Reply:
Done.